lib/network.go

```diff
@@ -5062,6 +5066,11 @@ func (tsl *TransactionSpendingLimit) ToMetamaskString(params *DeSoParams) string
 		indentationCounter--
 	}
 
+	// IsUnlimited
+	if tsl.IsUnlimited {
+		str += "FULL ACCESS"
+	}
```

Comment:
If it's full access, should we even include the spending limit object? seems a little unnecessary to have both

Reply:
Feels like if we don't indicate that it's full access, people might not realize how much permission this key has.

Reply:
It should certainly say full access, but we should leave out the other pieces of the spending limit object like the maps and the global DESO limit.